src/java.desktop/share/classes/sun/print/PSPrinterJob.java

```diff
@@ -393,11 +393,12 @@ private static Properties initProps() {
                 }
 
                 // Load property file
-                InputStream in =
-                    new BufferedInputStream(new FileInputStream(f.getPath()));
                 Properties props = new Properties();
-                props.load(in);
-                in.close();
+                try (var is = new FileInputStream(f.getPath());
+                     var bis = new BufferedInputStream(is))
```

Comment:
It will be better to use the same pattern everywhere, since the types are used in other places it will be good to use it here as well.

Reply:
as you wish